devices/licardofpv.json

```diff
@@ -0,0 +1,30 @@
+[
+  {
+    "category": "LicardoFPV",
+    "name": "Timer C3",
+    "targets": [
+      {
+        "flashingMethod": "UART",
+        "name": "esp32c3"
+      }
+    ],
+    "userDefines": [],
+    "wikiUrl": "https://space.bilibili.com/7403418",
```

Comment:
You got a real documentation page for this? A video should also work, prefer not to direct to a channel directly.

Reply:
removed